tests/test_operations.py

```diff
@@ -88,12 +88,18 @@ def test_interpolate(self):
         known_point = Point(1,1.5)
         interpolated_point = test_line.interpolate(.5, normalized=True)
         self.assertEqual(interpolated_point, known_point)
-        
-        # Issue #653; should raise ValueError on exception
+
+        # Issue #653; should nog segfault for empty geometries
         empty_line = loads('LINESTRING EMPTY')
-        assert(empty_line.is_empty)
-        with pytest.raises(ValueError):
-            empty_line.interpolate(.5, normalized=True)
-        
+        assert empty_line.is_empty
+        interpolated_point = empty_line.interpolate(.5, normalized=True)
+        assert interpolated_point.is_empty
```

Comment:
This test/behaviour comes from https://github.com/Toblerity/Shapely/issues/653. GEOS segfaults when passing an empty linestring, and Shapely solved this with the `exceptNull` decorator checking if any of the arguments was empty. PyGEOS currently also checks for this explicitly (to avoid the segfault, see https://github.com/pygeos/pygeos/blob/32397d998d2fc7fe3afef673539dace9fd8efe7f/src/geos.c#L297), but we decided to return an empty point instead of raising an error (see https://github.com/pygeos/pygeos/blob/32397d998d2fc7fe3afef673539dace9fd8efe7f/src/ufuncs.c#L373)

Any strong opinions on the desired behaviour?

cc @snorfalorpagus

Reply:
This change of behavior is welcome, particularly since this is what PostGIS does:
```
db=> SELECT ST_AsText(ST_LineInterpolatePoint('LINESTRING EMPTY'::geometry, 0.2));
  st_astext
-------------
 POINT EMPTY
(1 row)
```